### ISSUE TITLE
inject is not a function

### ISSUE THREAD

**Issue by lostpebble** [philmander/mobx-preact#4]
I'm busy trying to move over from React to Preact and aliasing this library. But it seems this is not fully compatible with the latest interface of the mobx-react library, as the new `@inject` syntax is not working, for injecting stores into a component.

And I just finished actually implementing the inject syntax on all my components because the old way, inside `@observer` was deprecated...

**Comment by nightwolfz** [comment]
Like the README says, this package is **_neither official, nor really maintained_**

Sorry :)

**Comment by lostpebble** [comment]
Fair enough. I've gone back to React for now, was struggling to get Preact to change over smoothly with my project. Will have to revisit sometime in the future again, might try make a pull request then.

**Comment by fa7ad** [comment]
@lostpebble 
just import `connect` aliased as `inject` and remove the @observer

React:  
```js
import {observer, inject} from 'mobx-react'

@inject(['abcstore']) @observer
class Something extends Component{...
```

Preact:
```js
import {connect as inject} from 'mobx-preact'

@inject(['abcstore'])
class Something extends Component{...
```

**Comment by philmander** [comment]
`inject` is now implemented (v1.0.0) as it is in `mobx-react`. `connect` can be used to inject and observer simultaneously.
